ui/src/components/Entity/EntityHeading.jsx

```diff
@@ -9,6 +10,7 @@ import 'components/common/ItemOverview.scss';
 class EntityHeading extends React.PureComponent {
   render() {
     const { entity, isProfile = false } = this.props;
+    const { value, unit } = selectUnit(new Date(parseInt(entity.lastViewed, 10)), Date.now());
```

Comment:
What happens if entity.lastViewed is undefined? maybe add a conditional to check before parsing

Reply:
Shoot, I had that previously, but removed it when refactoring, nice catch. I'll update this accordingly